### ISSUE TITLE
Cuda error: CUDA error: device-side assert triggered

### ISSUE THREAD

**Issue by Stefano-retinize** [rd20karim/M2T-Interpretable#2]
Hey @rd20karim thsnks for your help do far, I am having this error in the code line: [Line 78](https://github.com/rd20karim/M2T-Interpretable/blob/2d402d64007b581c21e496ee501571d2f3f71e23/train_wandb.py#L78)

It seems it has to do with the number of classes the model is predicting or something.

Maybe related with that is that I noticed that the vocab size inside vocabulary.vocab_size_unk is different than the KIT model input_dim. So, I guess somehow the vocabulary class is building a longer vocabulary than what the trained model have. 

**Is there any chance you can share the class vocabulary object maybe as a .pkl (for the models model_ton5mfwh and model_u8qatt2y)? Maybe this is faster, since I guess is going to be more difficult to spot what's the error when is building this vocabulary class.**

Thanks for your help.!

**Comment by rd20karim** [comment]
Hi @Stefano-retinize,
Did you download KIT-ML or HumanML3D dataset? what is the size of your current vocabulary.
Building the dataset should be done using only the scripts in the other [project code](https://github.com/rd20karim/M2T-Segmentation)

**Comment by Stefano-retinize** [comment]
Yeah I actually changed to the segmentation project and it all worked fine. I am now trying to figure out what each number means in the dataset. I think is all position based.

Thanks for all your help!
